frontend/src/features/frame/components/topbar/TopBar.tsx

```diff
@@ -15,7 +15,7 @@ import { BeforeInstallPromptEvent } from '../../pwaPrompt';
 
 // Allow to position contents like the footer relatively to the top of the
 // page.
-export const topBarHeight = 80;
+export const topBarHeight = 62;
```

Comment:
I think we also need to update the css override for notifications in "index.css":
https://github.com/tournesol-app/tournesol/blob/b39e6e5f4e6202da70318febd6d6fbaaebd72514/frontend/src/index.css#L37

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/1976/commits/135bea69fa3432baec7182089084890d3a222b6a